event-producer/main.py

```
@@ -0,0 +1,206 @@
+                  repository {
+                    nameWithOwner
+                  }
+                }
+              }
+            }
+          }
+        }
+        """
+        
+        variables = {"cursor": self.cursor} if self.cursor else {}
+        response = requests.post(
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)